config/runtime/fstests.jinja2

```diff
@@ -1,32 +1,60 @@
 {# -*- mode: Python -*- -#}
 {# SPDX-License-Identifier: LGPL-2.1-or-later -#}
- 
+
 {%- extends 'base/python.jinja2' %}
- 
+
 {%- block python_imports %}
 {{ super() }}
 import subprocess
 {%- endblock %}
- 
+
 {%- block python_globals %}
 {{ super() }}
 REVISION = {{ revision }}
 {% endblock %}
- 
-{%- block python_body %}
-def check_kvm_xfstest_env():
-   try:
-       result = subprocess.run("kvm-xfstests --help", check=True, shell=True)
-       if result.returncode == 0:
-           print(f"KVM-xftests found.")
-           return True
-       else:
-           return False
-   except Exception as e:
-       print(f"Raised exception: {e}")
-       return False
- 
-def main(argv):
-   print("Checking if KVM-xfstests is available...")
-   check_kvm_xfstest_env()
+
+{% block python_job -%}
+class Job(BaseJob):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def _kernel_config(self, kdir):
+        try:
+            os.chdir(kdir)
+            result = subprocess.run("kvm-xfstests install-kconfig", check=True, shell=True)
+            if result.returncode == 0:
+                if os.path.exists('.config'):
```

Comment:
Did you see a case where the command returned no error but the `.config` file was missing?

Reply:
yes, I did, but now I also included a message to "report" about just returning `False` is too vague.

Reply:
I think that's a bug in `kvm-xfstests` if the `install-kconfig` command doesn't generate the `.config` file and doesn't report an error either.  In fact the build would then fail, so I don't think it's necessary to check for the presence of this file.  It might also be empty, corrupt, set up for the wrong architecture etc.